File: spring-console-client/src/main/scala/com/mobilepetroleum/springconsole/client/console/EntryPoint.scala
package com.mobilepetroleum.springconsole.client.console

import org.slf4j.LoggerFactory

object EntryPoint {

  val logger = LoggerFactory.getLogger("console")

  def main(args: Array[String]) {
    try {
      new ConsoleApplication().start(args)
    } catch {
      case e: Exception => logger.error(s"${e.getMessage}", e)
    }
  }

}

File: spring-console-client/src/main/java/com/mobilepetroleum/springconsole/client/console/EntryPoint.java
package com.mobilepetroleum.springconsole.client.console;

import org.slf4j.Logger;
import org.slf4j.LoggerFactory;

public class EntryPoint {

    private final static Logger logger = LoggerFactory.getLogger("console");

    public static void main(String[] args) {
        try {
            new ConsoleApplication().start(args);
        } catch (Exception e) {
            logger.error(e.getMessage());
        }
    }

}
